fastestimator/dataset/data/horse2zebra.py

```diff
@@ -62,24 +61,21 @@ def load_data(root_dir: Optional[str] = None) -> Tuple[UnpairedDataset, Unpaired
             zip_file.extractall(root_dir)
         os.rename(os.path.join(root_dir, 'horse2zebra'), data_folder_path)
 
-    test_a = UnlabeledDirDataset(root_dir=os.path.join(data_folder_path, 'testA'),
-                                 data_key="A",
-                                 file_extension='.jpg',
-                                 recursive_search=False)
-    test_b = UnlabeledDirDataset(root_dir=os.path.join(data_folder_path, 'testB'),
-                                 data_key="B",
-                                 file_extension='.jpg',
-                                 recursive_search=False)
-    train_a = UnlabeledDirDataset(root_dir=os.path.join(data_folder_path, 'trainA'),
-                                  data_key="A",
-                                  file_extension='.jpg',
-                                  recursive_search=False)
-    train_b = UnlabeledDirDataset(root_dir=os.path.join(data_folder_path, 'trainB'),
-                                  data_key="B",
-                                  file_extension='.jpg',
-                                  recursive_search=False)
+    test_a = DirDataset(root_dir=os.path.join(data_folder_path, 'testA'),
+                        data_key="A",
+                        file_extension='.jpg',
+                        recursive_search=False)
+    test_b = DirDataset(root_dir=os.path.join(data_folder_path, 'testB'),
+                        data_key="B",
+                        file_extension='.jpg',
+                        recursive_search=False)
+    train_a = DirDataset(root_dir=os.path.join(data_folder_path, 'trainA'),
+                         data_key="A",
+                         file_extension='.jpg',
+                         recursive_search=False)
+    train_b = DirDataset(root_dir=os.path.join(data_folder_path, 'trainB'),
+                         data_key="B",
+                         file_extension='.jpg',
+                         recursive_search=False)
 
-    train = UnpairedDataset(train_a, train_b)
-    test = UnpairedDataset(test_a, test_b)
-
-    return train, test
+    return train_a, train_b, test_a, test_b
```

Comment:
Make these into BatchDatasets

Reply:
batch_size is required to use `BatchDatasets` in this case. It makes more sense to initialize `BatchDataset` under the `get_estimator` where user can leverage the `batch_size` argument.

Reply:
in that case lets make batch_size an input arg of the horse2zebra function

Reply:
done